projects/cm_mcu/MonitorI2CTask.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * MonitorI2CTask.h
+ *
+ *  Created on: June 30, 2022
+ *      Author: pkotamnives
+ */
+
+#ifndef PROJECTS_CM_MCU_MONITORI2CTASK_H_
+#define PROJECTS_CM_MCU_MONITORI2CTASK_H_
+
+#include "common/smbus.h"
+#include "semphr.h"
+#include "Tasks.h"
+
+extern SemaphoreHandle_t xMonSem;
```

Comment:
remind me what this is used for?

Reply:
it should be removed and I just did. A remnant from copying/paste code. Done.

Reply:
I think it is not even needed in MonitorTask.c? it doesn't seem to be used in anywhere.

Reply:
I think you are right!

Reply:
Done.